### ISSUE TITLE
DHT timed out while looking for peers - incomplete downloads keep retrying

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/lbry#1284]
## The Issue

If a client has an incomplete download, it will continue searching for peers in the background. Currently, this will log a warning each time this search attemps and times out. In this scenario (or all?), it should probably fail silently. 

```
2018-07-03 17:36:21,937 WARNING  lbrynet.dht.peerfinder:54: DHT timed out while looking peers for blob bcd8ec5861ebe403aa566e5566932daf15a5823dc2f0ed66403db7c9f36efcea405a5663c1e4e489c52fabc9f0ac30ce after 30 seconds.
2018-07-03 17:36:51,940 WARNING  lbrynet.dht.peerfinder:54: DHT timed out while looking peers for blob cb1de038e53d29fb42608627cbcdd1a68cd67a895937611c8064fa09854cfc8690a3f0b316466581bcfdcdc145cceb8b after 30 seconds.
2018-07-03 17:37:26,940 WARNING  lbrynet.dht.peerfinder:54: DHT timed out while looking peers for blob bcd8ec5861ebe403aa566e5566932daf15a5823dc2f0ed66403db7c9f36efcea405a5663c1e4e489c52fabc9f0ac30ce after 30 seconds.
2018-07-03 17:37:56,941 WARNING  lbrynet.dht.peerfinder:54: DHT timed out while looking peers for blob cb1de038e53d29fb42608627cbcdd1a68cd67a895937611c8064fa09854cfc8690a3f0b316466581bcfdcdc145cceb8b after 30 seconds.
2018-07-03 17:38:31,941 WARNING  lbrynet.dht.peerfinder:54: DHT timed out while looking peers for blob bcd8ec5861ebe403aa566e5566932daf15a5823dc2f0ed66403db7c9f36efcea405a5663c1e4e489c52fabc9f0ac30ce after 30 seconds.
2018-07-03 17:39:01,943 WARNING  lbrynet.dht.peerfinder:54: DHT timed out while looking peers for blob cb1de038e53d29fb42608627cbcdd1a68cd67a895937611c8064fa09854cfc8690a3f0b316466581bcfdcdc145cceb8b after 30 seconds.
2018-07-03 17:39:36,944 WARNING  lbrynet.dht.peerfinder:54: DHT timed out while looking peers for blob bcd8ec5861ebe403aa566e5566932daf15a5823dc2f0ed66403db7c9f36efcea405a5663c1e4e489c52fabc9f0ac30ce after 30 seconds.
2018-07-03 17:40:06,947 WARNING  lbrynet.dht.peerfinder:54: DHT timed out while looking peers for blob cb1de038e53d29fb42608627cbcdd1a68cd67a895937611c8064fa09854cfc8690a3f0b316466581bcfdcdc145cceb8b after 30 seconds.
2018-07-03 17:40:16,039 INFO     lbrynet.core.server.BlobRequestHandler:151: Sending 8a0d9c17a81f21a2 to 90.193.226.69:50484
2018-07-03 17:40:24,233 INFO     lbrynet.core.server.BlobRequestHandler:151: Sending 1133e614f0622ef5 to 90.193.226.69:50484
2018-07-03 17:40:34,036 INFO     lbrynet.core.server.BlobRequestHandler:151: Sending 85e7f3c740ef527c to 90.193.226.69:50484
2018-07-03 17:40:37,667 INFO     lbrynet.core.server.BlobRequestHandler:151: Sending 7be8ecf552368743 to 90.193.226.69:50484
2018-07-03 17:40:45,500 INFO     lbrynet.core.server.BlobRequestHandler:151: Sending 0e33a11eb5d6f51f to 90.193.226.69:50484
2018-07-03 17:40:54,085 INFO     lbrynet.core.server.BlobRequestHandler:151: Sending 7c90d19d7014e37e to 90.193.226.69:50484
2018-07-03 17:41:03,433 INFO     lbrynet.core.server.BlobRequestHandler:151: Sending 51d39fc81bc1bcf9 to 90.193.226.69:50484
2018-07-03 17:41:07,503 INFO     lbrynet.core.server.BlobRequestHandler:151: Sending 9a0c333745065bbf to 90.193.226.69:50484
2018-07-03 17:41:11,884 INFO     lbrynet.core.server.BlobRequestHandler:151: Sending 1b6c444cb65bebf4 to 90.193.226.69:50484
2018-07-03 17:41:13,540 WARNING  lbrynet.dht.peerfinder:54: DHT timed out while looking peers for blob bcd8ec5861ebe403aa566e5566932daf15a5823dc2f0ed66403db7c9f36efcea405a5663c1e4e489c52fabc9f0ac30ce after 30 seconds.
2018-07-03 17:41:18,486 INFO     lbrynet.core.server.BlobRequestHandler:151: Sending 0b8a89bfcc8c2e58 to 90.193.226.69:50484
2018-07-03 17:41:23,851 INFO     lbrynet.core.server.BlobRequestHandler:151: Sending 3071495c49c25c08 to 90.193.226.69:50484
2018-07-03 17:41:27,440 INFO     lbrynet.core.server.BlobRequestHandler:151: Sending 1ec0630d95caece3 to 90.193.226.69:50484
2018-07-03 17:41:31,937 INFO     lbrynet.core.server.BlobRequestHandler:151: Sending 6aa344caff022efc to 90.193.226.69:50484
2018-07-03 17:41:37,220 INFO     lbrynet.core.server.BlobRequestHandler:151: Sending 1858f1c7cb313400 to 90.193.226.69:50484
2018-07-03 17:41:42,253 INFO     lbrynet.core.server.BlobRequestHandler:151: Sending 996d664c4efecb6f to 90.193.226.69:50484
2018-07-03 17:41:43,542 WARNING  lbrynet.dht.peerfinder:54: DHT timed out while looking peers for blob cb1de038e53d29fb42608627cbcdd1a68cd67a895937611c8064fa09854cfc8690a3f0b316466581bcfdcdc145cceb8b after 30 seconds.
2018-07-03 17:41:46,532 INFO     lbrynet.core.server.BlobRequestHandler:151: Sending eadd043b9f547416 to 90.193.226.69:50484
2018-07-03 17:41:50,727 INFO     lbrynet.core.server.BlobRequestHandler:151: Sending b8f13d9aa0e191ea to 90.193.226.69:50484
2018-07-03 17:41:55,299 INFO     lbrynet.core.server.BlobRequestHandler:151: Sending de133702934145f9 to 90.193.226.69:50484
2018-07-03 17:41:59,378 INFO     lbrynet.core.server.BlobRequestHandler:151: Sending 15377d3c625ad05e to 90.193.226.69:50484
2018-07-03 17:42:05,006 INFO     lbrynet.core.server.BlobRequestHandler:151: Sending 764eaf57dd22aad6 to 90.193.226.69:50484
2018-07-03 17:42:09,908 INFO     lbrynet.core.server.BlobRequestHandler:151: Sending 93addcd136c7fe10 to 90.193.226.69:50484
2018-07-03 17:42:13,486 INFO     lbrynet.core.server.BlobRequestHandler:151: Sending 624a87abf914f4e8 to 90.193.226.69:50484
2018-07-03 17:42:18,151 INFO     lbrynet.core.server.BlobRequestHandler:151: Sending 8fcd30d0b81b7ecf to 90.193.226.69:50484
2018-07-03 17:42:18,543 WARNING  lbrynet.dht.peerfinder:54: DHT timed out while looking peers for blob bcd8ec5861ebe403aa566e5566932daf15a5823dc2f0ed66403db7c9f36efcea405a5663c1e4e489c52fabc9f0ac30ce after 30 seconds.

```
## System Configuration

 

- LBRY Daemon version:
- LBRY App version:
- LBRY Installation ID:
- Operating system:

**Comment by tzarebczan** [comment]
The frequency does slow down later on...maybe once per 10/20 minutes, will continue to monitor.

**Comment by tzarebczan** [comment]
So it does stop the warning, but the stuck downloads continue to retry in the background.

**Comment by tzarebczan** [comment]
Ran a verbose log to confirm that the daemon does keep trying to find peers. This could be what's causing some of the high cpu usage on some installs. 
Log:
https://gist.github.com/tzarebczan/88bb8b2d5511f3fb63150647e1fb8683

**Comment by tzarebczan** [comment]
This was fixed in 0.20.4
